tests/clv/models/test_basic.py

```diff
@@ -234,3 +234,19 @@ def test_model_config_warns(self) -> None:
         assert model.model_config == {
             "x": Prior("StudentT", mu=0, sigma=5, nu=15),
         }
+
+    def test_backward_compatibility_with_old_config(self):
+        old_model_config = {
+            "alpha_prior": Prior("Weibull", alpha=2, beta=10),
+            "r_prior": Prior("Weibull", alpha=2, beta=1),
+        }
+        model = CLVModelTest(model_config=old_model_config)
+        model.fit(tune=0, chains=2, draws=5)
+        model.save("old_model_config_test")
```

Comment:
This isn't testing for loading from an *old* saved model. The idata.posterior will have the suffix. We want to check for that suffix in the posterior will not break using the model in the future